maintainers/team-list.nix

```diff
@@ -56,11 +62,29 @@ with lib.maintainers; {
     scope = "Maintain BEAM-related packages and modules.";
   };
 
+  blockchains = {
+    members = [
+      mmahut
+      RaghavSood
+    ];
+    scope = "Team for feature-freeze-teams.pl, not to be used as package maintainers!";
```

Comment:
I wanted to take the fear of people that they will now get randomly added as the maintainers of packages because it's part of their subsystem.
Changing the scope later is never a problem and can freely be done if people want to use the team for other purposes.

Reply:
Setting someone else as maintainer without their approval is a big no-no anyway, so I don't see the problem.